control/config.go

```diff
@@ -75,6 +75,39 @@ type Config struct {
 	Plugins           *pluginConfig     `json:"plugins,omitempty"yaml:"plugins,omitempty"`
 }
 
+const (
+	CONFIG_CONSTRAINTS = `
+			"control" : {
+				"type": ["object", "null"],
+				"properties": {
+					"plugin_trust_level": {
+						"type": "integer",
+						"minimum": 0,
+						"maximum": 2
+					},
+					"auto_discover_path": {
+						"type": "string"
+					},
+					"cache_expiration": {
+						"type": "string"
+					},
+					"max_running_plugins": {
+						"type": "integer"
```

Comment:
Probably need a min here also. Do we interpret 0 to mean unlimited? Not sure if it should be 0 or 1.